src/scripts/main.js

```diff
@@ -1,3 +1,17 @@
 'use strict';
 
-// write your code here
+const populationArray = document.querySelectorAll('.population');
+const totalPopulation = document.querySelector('.total-population');
+const averagePopulation = document.querySelector('.average-population');
+
+const countriesCount = populationArray.length;
+let total = 0;
+
+populationArray.forEach((stringPopulation) => {
+  total += parseInt(stringPopulation.textContent.replace(/,/g, ''), 10);
+});
```

Comment:
Ensure that all elements with the class 'population' contain valid numeric values. If any element contains non-numeric text, `parseInt` will return `NaN`, which could lead to incorrect calculations. Consider adding error handling or validation to ensure that each element's text content is a valid number.